Client/.env

```diff
@@ -1,4 +1,6 @@
 REACT_APP_AUTH0_DOMAIN=dev-tumjlmbcbam83fd5.us.auth0.com
 REACT_APP_AUTH0_CLIENT_ID=kX8pQGaSE09v6KurOZmcAMJ20WUhv4Zd  
-# REACT_APP_SERVER_URL=http://143.110.184.87:3005/uploads
-REACT_APP_SERVER_URL=http://localhost:3005/uploads
\ No newline at end of file
+# REACT_APP_FETCH_URL=http://134.209.216.93:8000/apkinfo
```

Comment:
Add `.env` in  `.gitignore` and remove it from github, it should not be available publically.